src/reporters/udp_sender.js

```diff
@@ -148,7 +148,7 @@ export default class UDPSender {
     });
     this._reset();
 
-    return { err: false, numSpans: numSpans };
+    return Promise.resolve({ err: false, numSpans: numSpans });
```

Comment:
is there a Promise-based version of `this._client.send()`? If not, the promise needs to be created before calling `send()` and completed in the callback at L143